src/scripts/browserdeviceprofile.js

```diff
@@ -609,7 +595,7 @@ define(['browser'], function (browser) {
         if (canPlayHls() && browser.enableHlsAudio !== false) {
             profile.TranscodingProfiles.push({
                 // hlsjs, edge, and android all seem to require ts container
-                Container: !canPlayNativeHls() || browser.edge || browser.android ? 'ts' : 'aac',
+                Container: !canPlayNativeHls() || browser.edge || browser.android || browser.firefox ? 'ts' : 'aac',
```

Comment:
Is this to fix the weird FLAC support on Firefox?

Reply:
I think it's supposed to be fixed on FF in the next version.

Reply:
I believe it was a test of sorts that I forgot to remove and forgot the purpose of. I've removed it for now.